spec/config/schema.rb

```diff
@@ -32,5 +32,9 @@
     t.integer 'parent_id',                                           null: false
     t.integer 'kind',                                                null: false
     t.string 'list', array: true, limit: 3
+
+    t.index 'model', unique: true
+    t.index ['name', 'wheels'], unique: true
+    t.index ['doors'], unique: true, where: 'enabled = true'
```

Comment:
This seems to break the sqlite tests. Can it be adjusted to work agnostically of the database?